src/sentry/options/defaults.py

```diff
@@ -864,6 +864,13 @@
 # Relay should emit a usage metric to track total spans.
 register("relay.span-usage-metric", default=False, flags=FLAG_AUTOMATOR_MODIFIABLE)
 
+# Killswitch for the Relay cardinality limiter, one of `enabled`, `disabled`, `passive.`
+# In `passive` mode Relay's cardinality limiter is active but it does not enforce the limits.
+#
+# Note: To fully enabled the cardinality limiter the feature `organizations:relay-cardinality-limiter`
+# needs to be rolled out as well.
+register("relay.cardinality-limiter.mode", default=None, flags=FLAG_AUTOMATOR_MODIFIABLE)
```

Comment:
`None` should be fine because of:

```python
    for option in RELAY_OPTIONS:
        if (value := sentry.options.get(option)) is not None:
            options[option] = value
```